### PR TITLE
[Docs] Clarify measurement ID in `Optimizing: Third Party Libraries`

### PR DESCRIPTION
**Pull request by housseindjirdeh**
Minor changes to `Optimizing: Third Party Libraries` doc page to clarify which type of ID for Google Analytics (and GTM)

### CODE REVIEW

File: docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx [hunk 1]
```diff
@@ -193,7 +195,8 @@ The `GoogleAnalytics` component can be used to include [Google Analytics
 
 <AppOnly>
 
-To load Google Analytics for all routes, include the component directly in your root layout:
+To load Google Analytics for all routes, include the component directly in your root layout and pass
+in your measurement ID:
```
Comment (styfle) [inline docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx:199]: Should line 212 be changed from `GA-XYZ` to `G-XYZ`?

File: docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx [hunk 2]
```diff
@@ -229,7 +232,8 @@ export default function RootLayout({ children }) {
 
 <PagesOnly>
 
-To load Google Analytics for all routes, include the component directly in your custom `_app`:
+To load Google Analytics for all routes, include the component directly in your custom `_app` and
+pass in your measurement ID:
```
Comment (styfle) [inline docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx:236]: Same question here: Should line 245 and 261 be changed from `GA-XYZ` to `G-XYZ`?

File: docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx (hunk 1 above)
Comment (housseindjirdeh) [reply docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx:199]: Good catch, updated!

File: docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx (hunk 2 above)
Comment (housseindjirdeh) [reply docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx:236]: Updated 👍

**Review by styfle — APPROVED** [review]
Perfect, thanks! 🥇

**Pull request merged**